tests/test-runner/bin/zts-report.py.in

```diff
@@ -379,11 +380,31 @@ def process_results(pathname):
 
     return d
 
+class ListMaybesAction(argparse.Action):
+    def __init__(self,
+                 option_strings,
+                 dest="SUPPRESS",
+                 default="SUPPRESS",
+                 help="list flaky tests and exit"):
+        super().__init__(
+            option_strings=option_strings,
+            dest=dest,
+            default=default,
+            nargs=0,
+            help=help)
```

Comment:
There appears to be a [compatibility issue](http://build.zfsonlinux.org/builders/CentOS%207%20x86_64%20%28TEST%29/builds/17752/steps/shell_4/logs/summary) here with python 2.7.